aten/src/TH/THGenerateFloatType.h

```diff
@@ -2,6 +2,10 @@
 #error "You must define TH_GENERIC_FILE before including THGenerateFloatType.h"
 #endif
 
+// no BLAS support for half types
+#pragma push_macro("USE_BLAS")
```

Comment:
Why do you need this?  Shouldn't THBlas functions be guarded properly so they don't try to execute BLAS code on float16, so you never need to mess with this define?

Reply:
It isn't even in the right file :-P. Removing.